Makefile

```diff
@@ -65,14 +65,20 @@ bin/cdi:
 	$(Q)echo "Building $@..."; \
 	cd cmd/cdi; $(GO_BUILD) -o $(abspath $@) .
 
+bin/validate:
+	$(Q)echo "Building $@..."; \
+	cd cmd/validate; $(GO_BUILD) -o $(abspath $@) .
+
 #
 # vendoring targets
 #
-vendor: ## ensure all the go.mod/go.sum files are up-to-date including vendor/ directory
+.PHONY: vendor
+vendor: ## ensure all the go.mod/go.sum files are up-to-date
 	@$(Q)echo "Running $@..."
 	@$(GO) mod tidy
 	@$(GO) mod verify
 	@(cd ${ROOTDIR}/cmd/cdi && ${GO} mod tidy)
+	@(cd ${ROOTDIR}/cmd/validate && ${GO} mod tidy)
```

Comment:
Should we also do ${GO} mod verify also in the cmd subdirs ?

Reply:
Yes, that probably makes sense.

Reply:
Updated. I'm constructing the `mod-tidy` and `mod-verify` targets automatically now.